src/main/kotlin/com/coder/toolbox/CoderRemoteEnvironment.kt

```diff
@@ -78,7 +78,7 @@ class CoderRemoteEnvironment(
             workspace = workspace.copy(latestBuild = build)
             update(workspace, agent)
         },
-        Action(context.i18n.ptrl("Update"), enabled = { workspace.outdated }) {
+        Action(context.i18n.ptrl("Update and start"), enabled = { workspace.outdated }) {
```

Comment:
We use "Update and restart" on the web dashboard if the workspace is already in a `Running` state.


<img width="359" alt="image" src="https://github.com/user-attachments/assets/b010fc01-a05a-46c4-949f-dc5ac60b62b2" />

And "Update and start" if the workspace is `Stopped`

<img width="257" alt="image" src="https://github.com/user-attachments/assets/714dea01-2262-49bd-b18a-762cc75351f5" />


So, we need to handle both cases separately.

Also, instead of disabling, can we just hide the unavailable actions?

Reply:
Sure I think we can do both though I'm not sure how do I get to have a running workspace but outdated so that I can test and play with it.

In the meantime let me write two issues to capture the requests.

Reply:
Update: unavailable actions are now hidden:
![Screenshot 2025-03-12 at 22 12 53](https://github.com/user-attachments/assets/dfc8c37e-d256-4cda-8f2b-a7c599ce238a)
![Screenshot 2025-03-12 at 22 13 56](https://github.com/user-attachments/assets/baf5b994-5bd3-4491-9f64-db2620d78191)

Reply:
> I'm not sure how do I get to have a running workspace but outdated so that I can test and play with it.

I can help provide one. If you are an admin, you can switch your workspace to a previous template version, and that makes it "Update and restart"

Reply:
I'm not an admin, that would indeed simplify my testing.